services/dmx.md

```diff
@@ -0,0 +1,30 @@
+#  DMX
+
+A service that communicates using DMX512-A protocol. More at https://en.wikipedia.org/wiki/DMX512.
+
+    identifier: 0x11cf8c05
+    camel: dmx
+
+A DMX universe contains 512 channels stored as bytes. This is too large to expose as a single register. 
+Are pipes the way to go?
+
+## Registers
+
+    rw enabled?: bool @ 0x80
+
+Determines if the DMX bridge is active
+
+    rw terminate?: bool @ 0x80
+
+If available, terminate the DMX circuit.
+
+## Commands
+
+    command send @ 0x80 {
+        packet: pipe
+    }
+    pipe report packet {
+        channels: bytes
+    }
```

Comment:
I think you need this:

```
    command send # 0x80 {}
    report {
        data_port: pipe_pore
    }
    pipe command data_port {
        data: bytes
    }
```

not sure if it makes sense to specify "format" of `data_port`, since it's just bytes.

Reply:
(we don't specify format in modelrunner, iothub, tcp)